backend/src/main/scala/cromwell/backend/standard/StandardSyncExecutionActor.scala

```diff
@@ -180,7 +181,7 @@ class StandardSyncExecutionActor(val standardParams: StandardSyncExecutionActorP
   def jobFailingDecider: Decider = {
     case exception: Exception =>
       completionPromise.tryFailure(
-        new RuntimeException(s"${createAsyncRefName()} failed and didn't catch its exception.", exception))
+        new RuntimeException(s"${createAsyncRefName()} failed and didn't catch its exception.", exception) with NoStackTrace)
```

Comment:
Can we add some additional soothing here along the lines that "but don't worry, this has been caught now and will result in this workflow being marked as failed?"